README.md

```diff
@@ -2,92 +2,133 @@
 
+The Portal Network is comprised of multiple peer-to-peer networks which together provide the data and functionality necessary to expose the standard JSON-RPC API.  These networks are specially designed to ensure that clients participating in these networks can do so with minimal expendature of networking bandwidth, CPU, RAM, and HDD resources.
 
-The Portal Network will be comprised of one or more decentralized peer-to-peer networks which together provide the data and functionality necessary to expose the standard JSON-RPC API.  These networks are being specially designed to ensure that clients participating in these networks can do so with minimal expendature of networking bandwidth, CPU, RAM, and HDD resources.
+The term "Portal Client" is used to describe a piece of software that participates in these networks and typically exposes the standard JSON-RPC API
 
-The term "Portal Client" is used to describe a piece of software that participates in these networks and exposes the standard JSON-RPC API
 
 ## Motivation
 
-This effort is motivated by two overlapping goals.
+The Portal Network is focused on delivering reliable, lightweight, and decentralized access to the Ethereum protocol.
 
-### Full Functionality for Stateless Clients
+### Prior Work on the "Light Ethereum Subprotocol" (LES)
 
-The core Ethereum protocol is moving towards a "stateless" model of block verification.  Under this model a client will be able to fully verify the execution of a block using a witness.  Such a client would no longer need to keep or maintain any of the Ethereum "state" data.  Such a client is very valuable in the context of the core protocol, as it facilitates a cleaner merge of the Eth1 and Eth2 chains.  
+The term "light client" has historically refered to a client of the existing DevP2P based LES network.  This network is designed using a client/server architecture.  The LES network has a total capacity dictated by the number of "servers" on the network.  In order for this network to scale, the "server" capacity has to increase.  This also means that at any point in time the network has some total capacity which if exceeded will cause service degredation across the network.  Because of this the LES network is unreliable when operating near capacity.
 
-> Additional reading on why stateless is so important to the Eth1/Eth2 merge: https://dankradfeist.de/ethereum/2021/02/14/why-stateless.html)
 
-What is easy to overlook is that such a "stateless" client will be unable to do much else without additional infrastructure.  Specifically it would be unable to serve the vast majority of the JSON-RPC apis.  The Portal Network provides this additional infrastructure, allowing stateless clients to also expose the external APIs that support the web3 ecosystem.
+## Architecture
 
+The Portal Network is built upon on the [Discover V5 protocol](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md) and operates over the UDP transport.
```

Comment:
This doc calls it the "Node Discovery Protocol v5", though "discv5" is probably also a fine name given that previous paragraphs did not give devp2p and LES full names. Calling it "Discover v5 protocol" might cause some momentary confusion